ParisBNF/abb/BNFabb92.xml

```diff
@@ -0,0 +1,682 @@
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">Shelf-mark of the Library on the
+                                        spine</desc>
+                                </item>
+
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">Stamp of the Library on the inner
+                                        part of the cover</desc>
+                                </item>
+
+                                <item xml:id="e3">
+                                    <locus target="#1r #168v"/>
+                                    <desc type="StampExlibris">A circle stamp, with the name
+                                            <persName ref="PRS1071dAbbadi" role="owner"/> on it, is
+                                        stamped in the first and last endleaves.</desc>
+                                </item>
+
+                                <item xml:id="e4">
+                                    <locus target="#1r"/>
+                                    <desc>A handwritten insertion of a shelfmark</desc>
+                                    <q xml:lang="fr">n: 92 Antoine d'Abbadie </q>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="false" xml:id="binding">
+                                <decoNote xml:id="b1">European binding, rebound in <date when="1903"
+                                        >1903</date>.</decoNote>
+                                <decoNote xml:id="b2">According to <persName ref="PRS1071dAbbadi"
+                                        role="owner"/>, <cit>
+                                        <quote>Leaves almost all detached; without a case and almost
+                                            as dilapidated as <ref type="mss"
+                                                corresp="BNFabb66A BNFabb66B">the n.66</ref></quote>
+                                    </cit> (<cit>
+                                        <quote xml:lang="fr">feuillets presque tous détachés; sans
+                                            étui et presque aussi délabré que <ref type="mss"
+                                                corresp="BNFabb66A BNFabb66B">le n.66</ref></quote>
+                                    </cit>)</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        <accMat>Small stripe of <material key="paper">paper</material>: "92. Mazhafa
+                            hidar (Livre de Hidar)" <locus target="#1r"/>
+                        </accMat>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1400" notAfter="1600" source="lettering">15th or
```

Comment:
I believe evidence is used here, not source? Could you check the guidelines? Thank you!

Reply:
right!